prusti-tests/tests_old/verify/ui/failing-postcondition.stderr

```diff
@@ -1,7 +1,7 @@
 error: [Prusti: verification error] postcondition might not hold.
  --> $DIR/failing-postcondition.rs:8:12
   |
-8 | #[ensures="something_true() && false"]
+8 | #[ensures(something_true() && false)]
   |            ^^^^^^^^^^^^^^^^    ^^^^^
```

Comment:
Did not realise in the previous ones, but the position of the arrows should actually change in this case.